features/command_line/format_option.feature

```diff
@@ -72,15 +72,3 @@ Feature: `--format` option
         does something that is pending (PENDING: No reason given)
         does something that is skipped (PENDING: No reason given)
       """
-
-  Scenario: Multiple formats and output targets
-    When I run `rspec example_spec.rb --format progress --format documentation --out rspec.txt`
-    Then the output should contain ".F**"
-    And the file "rspec.txt" should contain:
-      """
-      something
-        does something that passes
-        does something that fails (FAILED - 1)
-        does something that is pending (PENDING: No reason given)
-        does something that is skipped (PENDING: No reason given)
-      """
```

Comment:
Er... If that's true we need to fix it, the point of this feature is that the first formatter uses the default output, and the second uses the rspec.txt file, changing that is a breaking change.